### ISSUE TITLE
label-content-name-mismatch is also a test for SC 1.3.1

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1471]
In addition to mapping to 4.1.2, label-content-name-mismatch should have the `wcag131` label.

**Comment by jkodu** [comment]
@WilcoFiers I believe you meant to say `aria-hidden-focus` based on this PR I see here - https://github.com/auto-wcag/auto-wcag/pull/449/files
